build.sbt

```diff
@@ -12,7 +12,7 @@ libraryDependencies ++= {
     "com.slack.api" % "slack-api-client" % "1.27.3",
     "args4j" % "args4j" % "2.33",
     "junit" % "junit" % "4.4" % Test,
-    "com.novocode" % "junit-interface" % "0.11" % Test exclude("junit", "junit-dep"),
+    "com.novocode" % "junit-interface" % "0.11" % Test exclude("junit", "junit-dep")
```

Comment:
This is allowed in Scala >= 2.12.2 (2017?). You might want to consider updating your builds.

Fine to remove this comma for BC

https://docs.scala-lang.org/sips/trailing-commas.html